### PR TITLE
added list of transfer methods

### PR DESCRIPTION
**Pull request by madhankumar3103**
added list of transfer methods

### CODE REVIEW

File: src/Hyperwallet/Hyperwallet.php [hunk 1]
```diff
@@ -2180,4 +2180,25 @@ public function listVenmoAccountStatusTransitions($userToken, $venmoAccountToken
         });
     }
 
+    /**
```
Comment (arao6) [inline src/Hyperwallet/Hyperwallet.php:2183]: Please update TransferMethod.php as well.

**Review by arao6 — APPROVED** [review]

**Review by Rrathinasabapath — APPROVED** [review]
Verified. Ready to be reviewed by Peter before merging.

**Comment by peter-joseph** [comment]
Merge conflict

**Review by arao6 — CHANGES_REQUESTED** [review]
Please pull the latest code from V4

File: src/Hyperwallet/Hyperwallet.php [hunk 2]
```diff
@@ -2398,4 +2396,27 @@ public function listBusinessStakeholders($userToken , $options) {
             return new BusinessStakeholder($entry);
         });
     }
+
+    //--------------------------------------
+    // Business Stakeholders
```
Comment (vkorotchenko) [inline src/Hyperwallet/Hyperwallet.php:2401]: was this moved here by mistake? seems like everything above is business stakeholders and below is transfer methods

**Review by vkorotchenko — COMMENTED** [review]
looks good. just one question

**Review by arao6 — APPROVED** [review]

File: src/Hyperwallet/Hyperwallet.php (hunk 2 above)
Comment (madhankumar3103) [reply src/Hyperwallet/Hyperwallet.php]: thanks, i changed the comments

**Review by vkorotchenko — APPROVED** [review]

**Pull request merged**